dom.bs

```diff
@@ -9877,47 +9840,11 @@ methods on {{Document}}.
 
 <h2 id=historical>Historical</h2>
 
-<p>As explained in <a href="#goals">goals</a> this standard is a significant revision of various
-obsolete DOM specifications. This section enumerates the incompatible changes.
-
-
-<h3 id=dom-events-changes>DOM Events</h3>
+<p id=domstringlist>Contrary to earlier revisions of this standard, {{DOMString}}, {{DOMException}},
+and {{DOMTimeStamp}} are now defined in Web IDL. {{DOMStringList}} is now defined in HTML.
+[[WEBIDL]] [[HTML]]
```

Comment:
I'd lean toward omitting these too, since they have a home that people can find without too much effort. It's the ones that totally disappeared that it seems nice to keep some record of. (But, no big deal either way.)